stats/collector.go

```diff
@@ -206,6 +213,20 @@ func (s *BufferedCollector) CollectMetadata(accountMetadata *Metadata) {
 	}
 }
 
+func (s *BufferedCollector) collectHeartbeat(ctx context.Context) {
+	// consistent id for all heartbeats of this run
+	const idLength = 16
+	uniqueRunID := randstr.Hex(idLength)
+	for {
+		select {
+		case <-time.After(heartbeatInterval):
```

Comment:
I would add a small jitter to avoid thundering herd from large installations

Reply:
Writes to this channel don't determine the time when events are sent, it is controlled by the `flushTicker`.
Large installation collects events from other sources too, so that problem will remain even if we add jitter here.
If you think it's a real issue I'll open a ticket and add jitter to the `flushTicker`.

Reply:
True.